Python/compile.c

```diff
@@ -161,6 +161,7 @@ struct compiler {
     int c_optimize;              /* optimization level */
     int c_interactive;           /* true if in interactive mode */
     int c_nestlevel;
+    int c_emit_bytecode;
```

Comment:
This special field requires a comment explaining its purpose. It seems to be specific to dead code. Maybe call "c_deadcode"? (I'm fine with c_emit_bytecode name.)

Reply:
Documented in the last amend.